docs/Rules/AvoidUsingAllowUnencryptedAuthentication.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Avoid sending credentials and secrets over unencrypted connections
+ms.custom: PSSA v1.22.0
+ms.date: 11/06/2022
+ms.topic: reference
+title: AvoidUsingAllowUnencryptedAuthentication
+---
+# AvoidUsingAllowUnencryptedAuthentication
+
+**Severity Level: Warning**
+
+## Description
+
+Avoid using the AllowUnencryptedAuthentication switch, which sends credentials and secrets over unencrypted connections.
+This should be avoided except for compatability with legacy systems.
```

Comment:
```suggestion
This should be avoided except for compatability with legacy systems. For more details, see https://learn.microsoft.com/powershell/module/microsoft.powershell.utility/invoke-webrequest#-allowunencryptedauthentication
```

Reply:
Added slightly different text in [c489ba4ed5c20a882463bce7c78d7e32444110db](https://github.com/PowerShell/PSScriptAnalyzer/pull/1857/commits/c489ba4ed5c20a882463bce7c78d7e32444110db) to include wording for other cmdlets as well.